migration-tool/src/test/java/software/amazon/awssdk/migration/recipe/ChangeAuthTypesTest.java

```diff
@@ -98,20 +102,53 @@ void defaultCredentialsProviderChain_getInstance_isRewritten() {
     void instanceCredentialsProvider_getInstance_isRewritten() {
         rewriteRun(
             java(
-                "import com.amazonaws.auth.InstanceProfileCredentialsProvider;\n"
+                "import com.amazonaws.auth.AWSCredentials;\n"
+                + "import com.amazonaws.auth.InstanceProfileCredentialsProvider;\n"
                 + "\n"
                 + "public class Example {\n"
                 + "    public static void main(String[] args) {\n"
                 + "        InstanceProfileCredentialsProvider credentials = InstanceProfileCredentialsProvider.getInstance();\n"
+                + "        AWSCredentials awsCredentials = credentials.getCredentials();\n"
                 + "    }\n"
                 + "}\n",
-                "import software.amazon.awssdk.auth.credentials.InstanceProfileCredentialsProvider;\n"
+                "import software.amazon.awssdk.auth.credentials.AwsCredentials;\n"
+                + "import software.amazon.awssdk.auth.credentials.InstanceProfileCredentialsProvider;\n"
                 + "\n"
                 + "public class Example {\n"
                 + "    public static void main(String[] args) {\n"
                 + "        InstanceProfileCredentialsProvider credentials = InstanceProfileCredentialsProvider.create();\n"
+                + "        AwsCredentials awsCredentials = credentials.resolveCredentials();\n"
                 + "    }\n"
-                + "}\n"
+                + "}"
+            )
+        );
+    }
+
+    @Test
+    @EnabledOnJre({JRE.JAVA_8})
+    void staticCredentials_getter_isRewritten() {
+        rewriteRun(
+            java("import com.amazonaws.auth.AWSCredentials;\n"
+                + "import com.amazonaws.auth.BasicSessionCredentials;\n"
+                + "\n"
+                + "public class Example {\n"
+                + "    public static void main(String[] args) {\n"
+                + "        BasicSessionCredentials credentials = new BasicSessionCredentials(\"foo\", \"bar\", \"123\");\n"
+                + "        String accessKeyId = credentials.getAWSAccessKeyId();\n"
+                + "        String secreteKey = credentials.getAWSSecretKey();\n"
+                + "        String sessionToken = credentials.getSessionToken();\n"
+                + "    }\n"
+                + "}\n",
+                "import software.amazon.awssdk.auth.credentials.AwsSessionCredentials;\n"
+                + "\n"
+                + "public class Example {\n"
+                + "    public static void main(String[] args) {\n"
+                + "        AwsSessionCredentials credentials = new AwsSessionCredentials(\"foo\", \"bar\", \"123\");\n"
+                + "        String accessKeyId = credentials.getAWSAccessKeyId();\n"
+                + "        String secreteKey = credentials.getAWSSecretKey();\n"
+                + "        String sessionToken = credentials.sessionToken();\n"
```

Comment:
Is this right? Seems like we're still calling getters the keys

Reply:
Oops, my bad, had unstaged changes in my local workspace. fixed!